include/open62541/plugin/certificategroup.h

```diff
@@ -36,7 +36,8 @@ struct UA_CertificateGroup {
 
     UA_StatusCode (*getRejectedList)(UA_CertificateGroup *certGroup, UA_ByteString **rejectedList, size_t *rejectedListSize);
 
-    UA_StatusCode (*verifyCertificate)(UA_CertificateGroup *certGroup, const UA_ByteString *certificate);
+    UA_StatusCode (*verifyCertificate)(UA_CertificateGroup *certGroup, const UA_ByteString *certificate,
+                                       const UA_ByteString *issuerCertificates, size_t issuerCertificatesSize);
```

Comment:
The whole idea of a certificate group is that it provides context for crypto operations.
The issuer list is part of that context.
So the issuer list does not need to be provided again. Remove.